src/components/common/Extensions/UserHub/partials/TransactionsTab/partials/TransactionsItem/TransactionsItem.tsx

```diff
@@ -68,67 +68,77 @@ const TransactionsItem: FC<TransactionsItemProps> = ({
               transition={{ duration: 0.4, ease: 'easeOut' }}
               className="overflow-hidden text-gray-600 text-md mt-2"
             >
-              {content.map((item, index) => (
-                <div
-                  key={item.key}
-                  className={clsx(styles.listItem, {
-                    'font-semibold': item.isCurrentAction,
-                    'before:bg-success-400':
-                      item.isCurrentAction &&
-                      (item.status === TRANSACTION_STATUSES.READY ||
-                        item.status === TRANSACTION_STATUSES.SUCCEEDED),
-                    'before:bg-negative-400':
-                      item.isCurrentAction &&
-                      item.status === TRANSACTION_STATUSES.FAILED,
-                    'before:!bg-blue-400':
-                      item.isCurrentAction && item.isPending,
-                  })}
-                >
-                  <div className="flex justify-between items-center">
-                    <h4>
-                      {index + 1}. {item.title}
-                    </h4>
-                    {item.status && !item.isPending && (
-                      <div
-                        className={clsx('flex ml-2', {
-                          'text-success-400':
-                            item.status === TRANSACTION_STATUSES.READY ||
-                            item.status === TRANSACTION_STATUSES.SUCCEEDED,
-                          'text-negative-400': TRANSACTION_STATUSES.FAILED,
-                        })}
-                      >
+              {content.map(
+                (
+                  {
+                    key,
+                    isCurrentAction,
+                    status: statusContent,
+                    isPending,
+                    notificationInfo,
+                  },
+                  index,
+                ) => (
+                  <div
+                    key={key}
+                    className={clsx(styles.listItem, {
+                      'font-semibold': isCurrentAction,
+                      'before:bg-success-400':
+                        isCurrentAction &&
+                        (statusContent === TRANSACTION_STATUSES.READY ||
+                          statusContent === TRANSACTION_STATUSES.SUCCEEDED),
+                      'before:bg-negative-400':
+                        isCurrentAction &&
+                        status === TRANSACTION_STATUSES.FAILED,
```

Comment:
status === TRANSACTION_STATUSES.FAILED  are you sure is that correct?

Reply:
I don't know. I've only destructured this object to be better readable, I didn't check conditionals.
![image](https://github.com/JoinColony/colonyCDapp/assets/119701775/95a62d95-87d3-467e-839c-b600a37351fb)
We can check it and fix it in a separate PR if it needed

Reply:
`statusContent` not `status`

Reply:
Fixed